client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -309,13 +317,15 @@ export default connect(
 		const { src, mimeType } = getImageEditorFileInfo( state );
 		const crop = getImageEditorCrop( state );
 		const isImageLoaded = isImageEditorImageLoaded( state );
+		const isGreaterThanMinimumDimensions = getImageEditorIsGreaterThanMinimumDimensions( state );
 
 		return {
 			src,
 			mimeType,
 			transform,
 			crop,
-			isImageLoaded
+			isImageLoaded,
+			isGreaterThanMinimumDimensions
```

Comment:
So this is optional, but we can also pass through showCrop instead of isGreaterThanMinimumDimensions
```jsx
showCrop: !! ( isImageLoaded && isGreaterThanMinimumDimensions )
```